CHANGELOG/CHANGELOG-4.1.md

```diff
@@ -2,6 +2,14 @@
 
 [Documentation](https://kubernetes-csi.github.io)
 
+# Changelog since v4.1.0
```

Comment:
Can you also add another line like `# v4.1.1` so that it's super obvious which release it's for? Same for `v4.1.0`

Reply:
Done